src/main/java/Storage/Storage.java

```diff
@@ -0,0 +1,64 @@
+package Storage;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Storage {
+
+    private static final Path FILE_PATH = Path.of("./save/tasks.txt");
+
+    /**
+     * Creates directory and tasks.txt if it does not exist
+     *
+     * @throws IOException If an I/O exception occurs during file handling
+     */
+    public static void createNewFile() throws IOException {
```

Comment:
Could we discuss the possibility of having a new Exception class for handling Storage class related exceptions?